index.js

```diff
@@ -82,6 +83,7 @@ function Bankai (entry, opts) {
     if (self.optimize) {
       b.transform(unassertify, { global: true })
       b.transform(yoyoify, { global: true })
+      b.transform(envify, { global: true })
```

Comment:
So, the changes would be

- no global
- set NODE_ENV to production so users don't have to do it themselves

something like

```js
b.transform(envify({
  NODE_ENV: 'production'
}))
```

is that correct? @yoshuawuyts @zigomir